src/components/Proposal/Create/SchemeForms/CreateContributionRewardProposal.tsx

```diff
@@ -375,7 +375,7 @@ class CreateContributionReward extends React.Component<IProps, IStateProps> {
                   </div>
                 </div>
 
-                <div className={css.reward}>
+                <div className={daoAvatarAddress === "0x519b70055af55a007110b4ff99b0ea33071c720a" ? css.hidden : css.reward}>
```

Comment:
Ouch this is ugly

In any case, DAO addresses should not be assumed unique across chains.  @orenyodfat